assembl/views/api/token.py

```diff
@@ -36,3 +40,13 @@ def auth_token(request, extra_headers=None):
     }
     token = encode_token(payload, request.registry.settings['session.secret'])
     return Response(token, 200, headers, content_type='text/plain')
+
+
+@view_config(route_name='check_password_token', request_method='GET',
+             renderer="json")
+def check_password_token(request):
```

Comment:
Correct. This is something we thought we might use, didn't, yet think we may still use someday. It allows the frontend to know that it is allowed to load on a closed discussion even if the user is not yet in the session.